.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,10 @@
+name: lint_python
```

Comment:
Perhaps name this and the file as a more generic `lint`?

Reply:
This codebase also includes JavaScript, Java, HTML, and CSS which can all also be linted.